src/MyEventEmitter.js

```diff
@@ -1,14 +1,65 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.listeners = {};
+  }
+
+  getListener(event) {
+    if (!this.isListenerExist(event)) {
+      this.listeners[event] = [];
+    }
+
+    return this.listeners[event];
+  }
+
+  isListenerExist(event) {
+    return this.listeners[event];
```

Comment:
The `isListenerExist` method should return a boolean value indicating whether the event exists. Consider changing this to `return !!this.listeners[event];` to ensure it returns a boolean.